docs/DIRECTORY_STRUCTURE.md

```diff
@@ -1,24 +1,27 @@
-# Project Directory Structure
+# Directory Structure
 
 Welcome to our project! This document outlines our project's directory structure, adhering to a "Feature-Driven Folder Structure" approach. The structure is inspired by insights from both an [English article](https://dev.to/profydev/screaming-architecture-evolution-of-a-react-folder-structure-4g25#indexjs-as-public-api) and a [Japanese article](https://zenn.dev/necscat/articles/d5d9b7a3f859d7).
 
-## Directory Structure Overview
+## Overview
 
-- [Project Directory Structure](#project-directory-structure)
-  - [Directory Structure Overview](#directory-structure-overview)
+- [Directory Structure](#directory-structure)
+  - [Overview](#overview)
     - [`app/`](#app)
+    - [`assets/`](#assets)
+    - [`assets/images/`](#assetsimages)
+    - [`assets/images/icons/`](#assetsimagesicons)
+    - [`assets/images/logos/`](#assetsimageslogos)
     - [`components/`](#components)
     - [`components/layouts/`](#componentslayouts)
-    - [`components/icons/`](#componentsicons)
     - [`components/pages/`](#componentspages)
```

Comment:
Minor point but it's `component/page` in actual directory structure.

Should we change the directory name at some point to meet with this document?

Reply:
Oh, I didn't notice the mismatch!

I think `components/pages/` is more natural. Sounds good?